core/croquet/src/main/java/org/lgna/croquet/views/HierarchyUtilities.java

```diff
@@ -57,168 +57,41 @@ private HierarchyUtilities() {
 
   public static final HowMuch DEFAULT_HOW_MUCH = HowMuch.COMPONENT_AND_DESCENDANTS;
```

Comment:
i did look at that, and marvel at the options that we were offering but never using. it would definitely uncomplicate things further to remove all of it, i just wasn't sure if it could ever be useful?  because adding it back in would be  annoying.